src/MoBi.Presentation/Tasks/SimulationUpdateTask.cs

```diff
@@ -82,33 +80,59 @@ public ICommand<IMoBiContext> UpdateSimulationFrom(IMoBiSimulation simulationToU
             fixedValueQuantities.AddRange(simulationToUpdate.Model.Root.GetAllChildren<IQuantity>(x => x.IsFixedValue));
          }
 
-         return updateSimulation(simulationToUpdate, templateBuildingBlock, buildConfigurationReferencingTemplate, configurationCommands, fixedValueQuantities);
+         return updateSimulation(simulationToUpdate, buildConfigurationReferencingTemplate, configurationCommands, templateBuildingBlock, fixedValueQuantities);
       }
 
-      private ICommand<IMoBiContext> updateSimulation(IMoBiSimulation simulationToUpdate, IBuildingBlock templateBuildingBlock, IMoBiBuildConfiguration buildConfigurationReferencingTemplates, IMoBiCommand configurationCommands, PathCache<IQuantity> fixedValueQuantities)
+      public ICommand ConfigureSimulation(IMoBiSimulation simulationToConfigure)
```

Comment:
The configuree simulation is exactly the same as update without the quick update